pubnub-kotlin/pubnub-kotlin-api/src/iosMain/kotlin/com/pubnub/kmp/Converters.kt

```diff
@@ -95,13 +95,18 @@ internal fun createPNChannelMembership(from: PubNubMembershipMetadataObjC): PNCh
 internal fun createPNMember(from: PubNubMembershipMetadataObjC): PNMember {
     return PNMember(
         uuid = from.uuid()?.let { createPNUUIDMetadata(from = it) },
-        custom = from.custom() as? Map<String, Any>, // TODO: Verify
-        updated = from.updated() ?: "",
-        eTag = from.eTag() ?: "",
+        custom = from.custom()?.safeCast(), // TODO: Verify
+        updated = from.updated().orEmpty(),
+        eTag = from.eTag().orEmpty(),
         status = from.status()
     )
 }
 
 internal inline fun <reified T, U> List<*>?.filterAndMap(mapper: (T) -> U): Collection<U> {
     return this?.filterIsInstance<T>()?.map(mapper) ?: emptyList()
+}
+
+@Suppress("UNCHECKED_CAST")
+inline fun <reified K, reified V> Map<*, *>.safeCast(): Map<K, V> {
+    return this as? Map<K,V> ?: emptyMap()
```

Comment:
I think at this stage I'd prefer if this was
`return this as? Map<K,V> ?: error("Cannot make the cast")`
so that we can easily find all places where this cast doesn't actually succeed when testing
I'm worried that if we silently return an empty map we might miss it

Reply:
Right, that makes sense. I'll fix it